openfisca_us/tests/policy/baseline/gov/states/md/tax/income/credits/md_poverty_line_credit.yaml

```diff
@@ -0,0 +1,9 @@
+name: Low income earner with 1000 in earned income
```

Comment:
Update name

Reply:
Should the test instead be:
![image](https://user-images.githubusercontent.com/16396180/179292440-7934fbdb-01f5-47ec-be13-bdb7ce172a40.png)